libclc/ptx-nvidiacl/lib/SOURCES

```diff
@@ -1 +1 @@
-mem_fence/fence.cl
+mem_fence/fence.cl
```

Comment:
can you revert your change, that's unnecessary

Reply:
i don't know why it happened. undone... thx